sdk/python/kfp/components/_data_passing.py

```diff
@@ -187,6 +187,8 @@ def serialize_value(value, type_name: str) -> str:
         str(type_name),
     ))
 
+
+# TODO: extract this out to a util module, likely outside either component or dsl.
```

Comment:
I'm not sure this function needs to be moved out of components. We're trying to keep the dependencies between modules organized.
Perhaps this function can just be called from this module.
Or it can just be copied - it's a simple function.

Reply:
The function is used in multiple places, and it doesn't belong to the data passing concept. Feels to me a util module would be more suitable.